Branch-SDK-Automation-TestBed/src/main/java/io/branch/saas/sdk/testbed/activities/MainActivity.java

```diff
@@ -138,6 +155,80 @@ public void onClick(View view) {
             Intent intent = new Intent(this, BUOReferenceActivity.class);
             intent.putExtra(Constants.TYPE, Constants.HANDLE_LINKS);
             startActivity(intent);
+        } else if (view == btSetDMAParams) {
+
+            getIntent().putExtra("branch_force_new_session", true);
+
+            Intent intent = new Intent(this, LogDataActivity.class);
+            intent.putExtra(Constants.TYPE, Constants.SET_DMA_Params);
+            String paramsString = (String) getIntent().getStringExtra("testData");
+
+            if (paramsString == null) {
+                intent.putExtra(Constants.STATUS, Constants.FAIL);
+                intent.putExtra(Constants.MESSAGE, "Failed to get input params. This test requires DMA related params set as intent extras.\n Example: getIntent().putExtra(\"testData\", \"{\\\"dma_eea\\\":\\\"Yes\\\",\\\"dma_ad_personalization\\\":\\\"Yes\\\",\\\"dma_ad_user_data\\\":\\\"Yes\\\",\\\"Include\\\":\\\"Yes\\\",\\\"URL\\\":\\\"https:\\\\/\\\\/timber.test-app.link\\\\/80OHAnv8DHb\\\"}\");");
+                startActivity(intent);
+            } else {
+                try {
+                    JSONObject jsonObject = new JSONObject(paramsString);
+                    String url = jsonObject.getString("URL");
+                    boolean eeaRegion = (jsonObject.getString("dma_eea").equalsIgnoreCase("yes"))? true:false;
+                    boolean adPersonalizationConsent = (jsonObject.getString("dma_ad_personalization").equalsIgnoreCase("yes"))? true:false;
```

Comment:
is there a constant for this string value? same for below

Reply:
No. These were only few values.So I didn't declare them as constants.